### PR TITLE
Housekeeping updates

### PR DESCRIPTION
**Pull request by prestist**

### CODE REVIEW

**Review by prestist — COMMENTED** [review]
@dustymabe  When you get the chance could I get a review on this ?

File: src/main.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export async function run(): Promise<void> {
 
     const issueBody = hydrateIssueTemplate(actionItems, meetingTopics)
     console.log('Create issue')
-    createThisReposIssue('Fedora CoreOS Weekly Meeting', issueBody)
+    createThisReposIssue(issueBody)
```
Comment (dustymabe) [inline src/main.ts:23]: are we somehow losing `Fedora CoreOS Weekly Meeting` in the issue title?

**Review by dustymabe — APPROVED** [review]
trivial LGTM

File: src/main.ts (hunk 1 above)
Comment (prestist) [reply src/main.ts:23]: Kinda; so this was an artifact of me not understanding actions well. I was passing a lot of values through via the go function signatures; but eventually I learned that you could run it via a config file. and so the "createissue" func was in a mixed state of using signatures and accepting the [config](https://github.com/coreos/fcos-meeting-action/blob/main/.github/workflows/create-meeting.yml) file. 

I can change it back to "Fedora CoreOS Weekly Meeting" but in this process I also did change the title to 'FCOS Meeting Checklist' ; thought is might be more appropriate?

Comment (dustymabe) [reply src/main.ts:23]: > I also did change the title to 'FCOS Meeting Checklist' ; thought is might be more appropriate?

+1 - works for me. You should be good to merge this.

**Pull request merged**